gke/README.md

```diff
@@ -25,7 +25,7 @@ You should have owner/editor role to perform the following tasks.
 
 ### One time setup
```

Comment:
Add the step to manually setup apigee? Feel like need to talk about that here but it also deserves a standalone setup guide..